std/process.d

```diff
@@ -1443,6 +1482,25 @@ int wait(Pid pid) @safe
     else version (Posix) assert (pid.osHandle < 0);
 }
 
+@system unittest // Pid.fromSystemPid()
+{
+    version (Windows)    TestScript prog = "exit %~1";
+    else version (Posix) TestScript prog = "exit $1";
+    auto pid = spawnProcess([prog.path, "10"]);
```

Comment:
This is racy - by the time you issue kill the process may already exit normally. An infinite loop would be a better choice then "exit $1"